CHANGELOG.md

```diff
@@ -16,6 +16,14 @@ Cromwell can be configured to send a POST request to a specified URL when a work
 includes the workflow id, terminal state, and (if applicable) final outputs or error message.
 See `WorkflowCallback` in [ReadTheDocs](https://cromwell.readthedocs.io/en/stable/) for more information.
 
+### Workflow runtime limit increased
```

Comment:
You get one thousand fake internet points for thinking to update the changelog! 

This file is consumed primarily by community Cromwell users who aren't Terra users, so I wouldn't mention Terra-specific concerns in here.